analytics_data_api/v0/views/learners.py

```diff
@@ -168,7 +181,7 @@ def _validate_query_params(self):
 
     def get_queryset(self):
         """
-        Fetches the user list from elasticsearch.  Note that an
+        Fetches the user list from elasticsearch. Note that an
```

Comment:
@lamagnifica: I'm not sure what plugin you're referring to. I *think* you're referring to the `trim_trailing_white_space_on_save` and `ensure_newline_at_eof_on_save` settings in Sublime, which shouldn't do anything to two-space-separated sentences. Can you show me this behavior at some point, so we can try to track down what's causing it?